src/argilla/client/feedback/unification.py

```diff
@@ -202,13 +204,11 @@ def unify_responses(self, records: List[FeedbackRecord], question: str):
 class RankingQuestionStrategy(RatingQuestionStrategyMixin, Enum):
     """
     Options:
-        - "mean": the mean value of the rankings
```

Comment:
hi, this is a breaking change and should be represented in the changelog and the docs. I do prefer to still include this to avoid having too much fragmentation in the unification methods.

![image](https://github.com/argilla-io/argilla/assets/25269220/61aa7383-e83f-4d30-830e-eae53d9f6e4b)

Reply:
```python
def calculate_average_ranking(data):
    label_rank_sum = {}
    label_count = {}

    for ranking in data:
        for item in ranking:
            label, rank = item.popitem()
            label_rank_sum[label] = label_rank_sum.get(label, 0) + rank
            label_count[label] = label_count.get(label, 0) + 1

    average_ranking = {label: label_rank_sum[label] / label_count[label] for label in label_rank_sum}

    return average_ranking

# Example usage:
data = [[{"label_1": 2}, {"label_2": 1}], [{"label_1": 1}, {"label_2": 2}]]

result = calculate_average_ranking(data)
print(result)
```

Reply:
after this I would only expect the labels to be mapped back to the original available ranks through an zip(rank, sorted_result_based_on_values)